src/alpaka_job_coverage/filter_backend_version.py

```diff
@@ -124,6 +124,11 @@ def compiler_backend_filter(row: List) -> bool:
             row, DEVICE_COMPILER, "<=", "15"
         ) and row_check_backend_version(row, ALPAKA_ACC_GPU_CUDA_ENABLE, ">", "11.5"):
             return False
+        
+        if row_check_version(
+            row, DEVICE_COMPILER, "<=", "16"
+        ) and row_check_backend_version(row, ALPAKA_ACC_GPU_CUDA_ENABLE, ">", "11.5"):
```

Comment:
Where are you get the information? Axel's list says, that CUDA 11.8 is supported: https://gist.github.com/ax3l/9489132

Reply:
I see, the official documentation mentioned CUDA 11.5 https://releases.llvm.org/16.0.0/docs/CompileCudaWithLLVM.html

I would prefer CUDA 11.8, because it is in the source code.

Reply:
CUDA 11.8 is only partially supported by clang. I opted for CUDA 11.5 because that is fully supported. https://github.com/llvm/llvm-project/blob/185b81e034ba60081023b6e59504dfffb560f3e3/clang/include/clang/Basic/Cuda.h#L40

Reply:
fair enough